rectools/models/sasrec.py

```diff
@@ -628,20 +630,20 @@ class SequenceDataset(TorchDataset):
 
     Parameters
     ----------
-    sessions: List[List[int]]
+    sessions: tp.List[tp.List[int]]
         User sessions in the form of sequences of items ids.
-    weights: List[List[float]]
+    weights: tp.List[tp.List[float]]
         Weight of each interaction from the session.
     """
 
-    def __init__(self, sessions: List[List[int]], weights: List[List[float]]):
+    def __init__(self, sessions: tp.List[tp.List[int]], weights: tp.List[tp.List[float]]):
```

Comment:
I think we can add some aliases like `Session = tp.List[int]`, `SessionWeights = tp.List[float]`, etc.  
Should be easier to read

Reply:
I wouldn't do that. We have sessions as Lists of Lists of ints and we have Sessions as torch.Tensors. So here I would leave it as it is now. It is more clear which data we are using right now.